microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -6,26 +6,26 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 07/06/2022
+ms.date: 08/03/2022
 ---
 # WebView2 API Reference
 
 The Microsoft Edge WebView2 control enables you to host web content in your application using Chromium-based Microsoft Edge as the rendering engine.
 
 WebView2 is available for the following frameworks (or platforms) and programming languages:
 
-* .NET
-   * [Core/C#](/dotnet/api/microsoft.web.webview2.core)
+*  .NET
+   * [Core](/dotnet/api/microsoft.web.webview2.core)
    * [WPF](/dotnet/api/microsoft.web.webview2.wpf)
    * [Windows Forms](/dotnet/api/microsoft.web.webview2.winforms)
 
-* WinRT for WinUI 2 (UWP)
-   * [Core/C#](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
+*  WinRT
+   * [Core](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/index)
    * [COM Interop/C++](/microsoft-edge/webview2/reference/winrt/interop/index)
-
-* WinRT for WinUI 3 (Windows App SDK)
-   * [Microsoft.UI.Xaml.Controls.WebView2 Class](/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.webview2)
-      * [Spec: WebView2 Xaml control](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/WebView2/WebView2_spec.md)<!-- changing master to main doesn't work 5/19/2022 -->
+   *  WinUI 2 (UWP)
+      *  Microsoft.UI.Xaml.Controls.WebView2 Class - TODO: need correct Ref URL if exists
```

Comment:
Looks like the correct URI should be:

https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.webview2?view=winui-2.8